test/stresstest/server.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
+import os.path
+import config
+
+class Server:
```

Comment:
Ah the equivalent to broker on top. I was wondering, might it also make sense - instead/in addtion to building the servers, to have way where they just pull (not build) the repsective containers and using them (then you could measure what is really released).

if they are started (assuming docker) with --net=host (=reachable by external testcleint without friuther forwarding) and --rm (do not leave old containers around) might be good

But this is not an "issue" with this PR, rather an idea :D

Reply:
This would be a good idea

Reply:
Should this be tackled in this PR or in a separate? I separated the stresstest script now from the scripts to run broker and server. I documented this in the README too. That you ahve to run the broker script and then run the stresstest script. In the run scripts for broker and server a flag could be set to distinguish between building from scratch and using the docker images.

Reply:
This will be tackled in a separate PR see #412